net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_REQ_USE=( sqlite )
+
+inherit distutils-r1 eutils python-r1
```

Comment:
No need to inherit python when inheriting distutils

Reply:
Done, thanks